core/services/ardupilot_manager/autopilot_manager.py

```diff
@@ -662,3 +670,23 @@ async def install_firmware_from_url(
 
     async def restore_default_firmware(self, board: FlightController) -> None:
         await self.firmware_manager.restore_default_firmware(board)
+
+    def set_unmanaged_board_master_endpoint(self, endpoint: Endpoint) -> bool:
+        self.configuration["unmanaged_board_master_endpoint"] = endpoint.as_dict()
+        self.settings.save(self.configuration)
+        return True
+
+    def get_unmanaged_board_master_endpoint(self) -> Endpoint:
+        default_master_endpoint = Endpoint(
+            name="Unmanaged Board Master Endpoint",
+            owner=self.settings.app_name,
+            connection_type=EndpointType.UDPServer,
+            place="0.0.0.0",
+            argument=14550,
```

Comment:
I think changing this to avoid conflict with our default `GCS Server Link`

```suggestion
            argument=14551,
```